### PR TITLE
Your Knowledge fields use React component and are saved with entry

### PR DESCRIPTION
**Pull request by brew**
The two 'Your Knowledge' fields now reuse the `QuestionForm` and likert question-type React components. The data for the question form is hardcoded into `EntryForm.jsx` as these are invariable questions asked about all datasets. The form will retrieve the question state from the component and save it to a new `aboutYouAnswers` column in the entry table, in the same way as for the main questionset.

### CODE REVIEW

File: census/controllers/census.js [hunk 1]
```diff
@@ -97,18 +97,21 @@ var submitPost = function(req, res, data) {
...
-                          '<a href="/place/%s/%s">See the queued submission</a>',
+                          '<a href="/place/%s/%s">See the queued submission</a>.',
                           current.place, req.params.year);
     errors.push({
       param: 'conflict',
       msg: msg
     });
   }
 
+  // errors.push({param: 'fake', msg: 'My fake error'});
```
Comment (nightsh) [inline census/controllers/census.js]: Why is this here? If it only helped during development, it should be removed.

File: census/migrations/20161027095620-add-entry-aboutyouanswers.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.addColumn('entry', 'aboutYouAnswers',
+      {
+        type: Sequelize.JSONB,
+        allowNull: true,
```
Comment (nightsh) [inline census/migrations/20161027095620-add-entry-aboutyouanswers.js]: Not sure about the use case, so double checking: is it allowed to be `null`?

File: census/ui_app/EntryForm.jsx [hunk 3]
```diff
@@ -48,119 +55,94 @@ const EntryForm = React.createClass({
...
-      </div>
-      <div className="comments">
-        <label htmlFor="yourKnowledgeOpenDataComment">Comments</label>
-        <textarea placeholder="Add comments"
-                  id="yourKnowledgeOpenDataComment"
-                  name="yourKnowledgeOpenDataComment"
-                  rows="5"
-                  defaultValue={this.props.answers.yourKnowledgeOpenDataComment}></textarea>
-      </div>
-    </div>
```
Comment (nightsh) [inline census/ui_app/EntryForm.jsx]: Removed a big chunk of markup, nice!

Comment (brew) [inline census/ui_app/EntryForm.jsx]: Yeah, it was becoming a pain to maintain.

File: census/migrations/20161027095620-add-entry-aboutyouanswers.js (hunk 2 above)
Comment (brew) [inline census/migrations/20161027095620-add-entry-aboutyouanswers.js]: I think it's okay. In reality it won't be for new entries, but we have a lot of legacy entries that don't include this data.

File: census/controllers/census.js (hunk 1 above)
Comment (brew) [inline census/controllers/census.js]: It's so useful :(
